.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: bash
+addons:
+  apt:
+    sources:
+      - debian-sid
+    packages:
+      - shellcheck
```

Comment:
This was copied from the [documentation][1].


[1]: https://github.com/koalaman/shellcheck#travis-ci-setup